docs/serving/graph-ha-cfg.md

```diff
@@ -32,9 +36,15 @@ graph structure and should be calculated accordingly. The following sections exp
 
 A consumer function is essentially a Nuclio function with a stream trigger. As part of the trigger, you can set a consumer group.  
 
-When the consumer function is part of a graph then the consumer function’s number of replicas is derived from the number of shards and is 
-therefore nonconfigurable. The same applies to the number of workers in each replica, which is set to 1 and is not configurable.  
-
+When the consumer function is part of a graph then the number of replicas per function depends on the  source:
```

Comment:
I don't think this is specific to the function being "a part of a graph" (meaning a multi-function graph?). Rather, it's always the case that we applies this logic to control the number of replicas.

Reply:
updated